config/frr/speaker-patch.yaml

```diff
@@ -78,6 +89,16 @@ spec:
                 attempts=$(( $attempts + 1 ))
               done
               tail -f /etc/frr/frr.log
+          livenessProbe:
+            exec:
+              command: ["/etc/frr_liveness/liveness.sh"]
+            periodSeconds: 5
+            failureThreshold: 3
+          startupProbe:
+            exec:
+              command: ["/etc/frr_liveness/liveness.sh"]
+            failureThreshold: 30
```

Comment:
is it 30 on purpose here? or should it be 3?

Reply:
it's on purpouse